ccd-definition/ComplexTypes/PayingExtraInfo.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "ID": "PayingExtraInfo",
```

Comment:
please rename this file to PayingExtraInfoLRspec in line with other LR file names.